engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -390,8 +390,10 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
           } else {
             List.empty
           }
-          // This .get is ok because we're already in a try/catch.
```

Comment:
This is actually still true, but maybe we can remove the `Try` now, if nothing else in this method looks dangerous?